### ISSUE TITLE
Some timelines / TTS do not work.

### ISSUE THREAD

**Issue by Crescr** [quisquous/cactbot#4509]
The following timelines and TTS do not work at all.
I feel that cactbot doesn't recognize these two contents.
・The Final Day
・The Mothercrystal（FACE）

The language of the client, ACT plug-in, and cactbot is set to Japanese.
Timelines other than the above two, including Extreme, and TTS work fine.

FFXIV_ACT_Plugin.dll　2.6.5.1
CactbotOverlay.dll　0.26.8.0
Overlay\OverlayPlugin.dll　0.18.2.0

Overlay Url gives the same result with either.
・https://quisquous.github.io/cactbot/ui/raidboss/raidboss.html
・<local>\cactbot\ui\raidboss\raidboss.html

**Comment by valarnin** [comment]
Can you provide the network log file for this, or test it on [raidemulator](https://quisquous.github.io/cactbot/ui/raidboss/raidemulator.html?OVERLAY_WS=ws://127.0.0.1:10501/ws) (make sure ACT is running and overlayplugin's websocket is enabled so raidemulator can read your cactbot config)?

**Comment by MaikoTan** [comment]
~~フェイスはFACEじゃなくて、Faith(信頼、信用)のほうが正しいと思います。それに、FF海外版にはTrust Systemと呼びます。~~

Can't reproduce. I tried cactbot on my side with the Japanese game client, but timeline and TTS works well in those dungeons that you mentioned above. Can you provide some log files for this?

My test environment:
Game Language: Japanese
ACT: 3.6.0 release 275
FFXIV_ACT_Plugin.dll: 2.6.5.1
OverlayPlugin.dll: 0.18.2.0
CactbotOverlay.dll: 0.26.8.0
And I am using the online version of overlay (https://quisquous.github.io/cactbot/ui/raidboss/raidboss.html)

**Comment by Crescr** [comment]
I have deleted the Final Day log, so I will attach only the Mother crystal for the time being.

We tested the actual combat and emulator with the following settings.
raidboss url : https://quisquous.github.io/cactbot/ui/raidboss/raidboss.html?OVERLAY_WS=wss://127.0.0.1:10501/ws&HOST_PORT=ws://127.0.0.1/fake/
raidemulator url : https://quisquous.github.io/cactbot/ui/raidboss/raidemulator.html?OVERLAY_WS=wss://127.0.0.1:10501/ws&HOST_PORT=ws://127.0.0.1/fake/

Actual Combat: Neither the timeline nor TTS works.
Emulator: Both timeline and TTS work fine.[
[Network_26501_20220522.log](https://github.com/quisquous/cactbot/files/8748847/Network_26501_20220522.log)
](url)

**Comment by Crescr** [comment]
I tested the emulator and both the timeline and TTS worked fine.

I found out why it didn't work.

It does not work only when playing the above two contents with a character without a clear flag (first time when the main quest is in progress).
At this time, the result is the same even if you leave and re-enter.
Both contents worked fine when I cleared it once and then replayed it immediately.
It is mysterious why only these two do not work at the first play.

The attached log is for the character who played The Mothercrystal for the first time in the quest.
[Network_26501_20220522.log](https://github.com/quisquous/cactbot/files/8749199/Network_26501_20220522.log)
